### ISSUE TITLE
nVidia Divers support macOS Catalina ?

### ISSUE THREAD

**Issue by wyatt-wong** [Benjamin-Dobell/nvidia-update#36]
Does this nVidia Drivers support macOS Catalina ? I got the following errors when I download and install the drivers.

<img width="682" alt="Screen Shot 2020-01-28 at 12 16 38 PM" src="https://user-images.githubusercontent.com/10095648/73235164-2cda7380-41c8-11ea-8546-b37358d71586.png">

**Comment by algol68** [comment]
1 - disable SIP
         restart in recovery mode
        type `csrutil disable`
2 - disable Gatekeeper
         back in 'normal' mode open Terminal and type `sudo spctl --master-disable` 
3 - remount file system (enable write mode of system volume)
         `sudo mount -uw /`
4 - force Finder process to restart
         `killall Finder`

you should be able to install nvidia drivers now ;-)

**Comment by farfromthe** [comment]
This works, can probably close this issue now. Thanks!!

**Comment by wyatt-wong** [comment]
Can we enable SIP after finish install the display drives in Catalina ?

Sent from my iPhone 11 Pro Max

On 26 Mar 2020, at 1:03 PM, farfromthe <notifications@github.com> wrote:

﻿
This works, can probably close this issue now. Thanks!!

**Comment by Benjamin-Dobell** [comment]
Just to clarify, nVidia's drivers do _not_ work on Catalina (or Mojave for that matter).

Yes, they'll install, and that may allow you to set some different screen resolutions, however macOS will _not_ be hardware accelerated.

There's been a lot of coverage on this issue, so a quick Google will explain what's going on. It's for Apple and nVidia to sort out; which they likely won't.

**Comment by wyatt-wong** [comment]
Well even without hardware acceleration, we can at least install the display drivers for Catalina

Sent from my iPhone 11 Pro Max

On 26 Mar 2020, at 2:22 PM, Benjamin Dobell <notifications@github.com> wrote:

﻿
Just to clarify, nVidia's drivers do not work on Catalina (or Mojave for that matter).

Yes, they'll install, and that may allow you to set some different screen resolutions, however macOS will not be hardware accelerated.

There's been a lot of coverage on this issue, so a quick Google will explain what's going on. It's for Apple and nVidia to sort out; which they likely won't.

**Comment by algol68** [comment]
agreed. workaround gets you past installation errors however drivers really don't perform as they should. @Benjamin-Dobell, i think noting this on the project homepage will be helpful (don't upgrade past High Sierra)

**Comment by MotorCityCobra** [comment]
@Benjamin-Dobell 
Do you think there is any hope for Big Sur?

**Comment by 23d1** [comment]
There is ZERO hope for Big Sur or any iteration of Apple Software when it comes to Nvidia. It's ridiculous, but it is what it is. Redshift is coming to Apple Metal. Octane X is already out. And so on and so forth.

**Comment by MotorCityCobra** [comment]
@23d1 
The latest I can go with the @Benjamin-Dobell 's drivers is High Sierra?

**Comment by NeverTheLess42** [comment]
Pimped a MacPro 3,1 with a nvidia 980Ti (6GB VRAM) and macOS Catalina. Connected it to the Samsung C49HGx 32:9 4K Monitor and got it to work like this:
- Connected the Monitor with HDMI and 1920x1080@144Hz
- Used the tipps from @algol68 (disable SIP, disable Gatekeeper, remounted root with -uw...)
- Ran Benjamins script to install the nvidia drivers.
- Important: after reboot, set the screen refresh rate to 60Hz, so higher resolutions are available.
- Installed the Resolution Helper Tool SwitchResX and defined a 3840x1080@60Hz resolution.
- Activated the resolution and voilà! Got it running...

Acceleration in macOS is not so important for me, I need the widescreen resolution for working - for gaming I reboot my MacPro 3,1 to Windows 10 Pro 20H2...

![Bildschirmfoto 2021-01-24 um 23 20 47](https://user-images.githubusercontent.com/69899028/105645278-ea12b080-5e9a-11eb-8b34-f1d90f72ec09.png)
